package.json

```diff
@@ -246,6 +251,10 @@
         {
           "command": "PowerShell.RefreshCommandsExplorer",
           "when": "config.powershell.sideBar.CommandExplorerVisibility"
+        },
+        {
+          "command": "PowerShell.InvokeRegisteredEditorCommand",
+          "when": "false"
```

Comment:
@TylerLeonhardt As far as I can tell this means that the command will _never_ be shown in the command palette...so this PR didn't do what it intended. 🧐

Reply:
FYI @rjmholt

Reply:
> @TylerLeonhardt As far as I can tell this means that the command will _never_ be shown in the command palette...so this PR didn't do what it intended. 🧐

I believe it's just the description that is off. This PR enabled it's usage in keybinds, e.g. this failed before it:

```json
{
    "key": "alt+shift+s",
    "command": "PowerShell.InvokeRegisteredEditorCommand",
    "args": { "commandName": "ConvertTo-SplatExpression" },
    "when": "editorLangId == 'powershell'"
},
```

Since it has parameters, and "Show Additional Commands from PowerShell Modules" fills the command palette role, I believe this was intended.